frontend/src/features/workspace/components/WorkspaceFormRow/WorkspaceRowsContext.tsx

```diff
@@ -65,6 +76,11 @@ export const WorkspaceRowsProvider = ({
         onClose={shareFormModalDisclosure.onClose}
         isFormPrivate={activeFormMeta?.status === FormStatus.Private}
       />
+      <CollaboratorModal
+        isOpen={collabModalDisclosure.isOpen}
+        formId={activeFormMeta?._id}
+        onClose={collabModalDisclosure.onClose}
+      />
```

Comment:
to ensure formId is always provided before rendering collaborator modal, we could use a conditional render:

```tsx
{activeFormMeta?._id && <CollaboratorModal
        isOpen={collabModalDisclosure.isOpen}
        formId={activeFormMeta._id} // guaranteed to exist when rendering 
        onClose={collabModalDisclosure.onClose}
      />}
```